### PR TITLE
Update productstoavoid.md and add manjaro to the bad list :)

### PR DESCRIPTION
**Pull request by IThundxr**
Manjaro is generally bad https://manjarno.snorlax.sh/ read there its extensive but not a good OS and ive added alot of stuff to why people shouldnt use it or reccomend it 

Maintainers Before editing anything here do let me know first

### CODE REVIEW

**Comment by Evernow** [comment]
Going to paste the conversation that occured on discord about this:

While I appreciate the contribution and agree that Manjaro has its problems, I disagree with most of the points on the page tbh

> SSL Certificates

Everyone forgets to renew subscriptions, especially those that require setting up a bot to auto renew, this page also doesn't mention that the site and repos were down for less than a day. It's not like this is a paid product, people receive that they receive for free. 

While the turn clock back wasn't that good of a suggestion, I disagree it's that big of a deal, not like you're turning back the clock for a year, it's for a day, and this happened some 8 years ago. 

> Holding packages back

For the stability argument, this does work. Plenty of distributions do this like Debian, Fedora, Arch (to a lesser extent with their testing repo, similar to Manjaro), RHEL (via CentOS Stream) and plenty of more distros. And it has catched problems that would have otherwise gone through, I remember a serious issue with bad firmware on Samsung SSDs, it was catched in the time it was on Manjaro testing branch, and as such did not affect the majority of Manjaro users.

As for the security argument of this, I do agree that Manjaro can improve on this, but in the worst case usually packages are only held back for a couple weeks. Do note that in the time I used to frequent the forums, Manjaro did ship security updates even if they didn't ship the entire stable update with an update, it's just that they didnt really announce that they did, you had to look for it in the update logs, so the main improvement they can do is improve the visibility of these updates.

> Stability

This is very situational, I used Manjaro for years and never had a single issue that wasn't my fault, while others have likely other experiences, it's pointless to share personal experiences, as everyone uses their OS differently.

What I will share is just pure data to back up the claim, for every testing and stable branch update Manjaro has a poll on whether the update went well, and the vast majoity of times the percentage of people reporting on the forums is over 90% without any issues, here's an example with the latest update today: https://forum.manjaro.org/t/stable-update-2023-02-12-kernels-openssl-nvidia-mesa-libreoffice-7-5-pamac-thunderbird/133923

92% say no problems as of typing this, and take into account that you're unlikely to come to the forums if you have no problems.

> The AUR

The first DDOS was their fault, the second one was fault of the AUR maintainer who recommended the behavior Pamac did, I can try finding a source for this later if you want it.

As for compatibility, I honestly never had an issue tbh, but then again it's situational.

> Rushing Asahi out the door

Don't know much about this and don't care (Fuck Apple), all I know is that the problems occured with a Manjaro image that they declared was unstable, so, isn't that the entire point? What is the point of unstable branches if not to test unstable changes?

> Their system update script used to run rm on the lockfile mid-transaction. 

As someone who volunteered on the forums, I can state that this behavior was justified, because this was by far one of the most common issues, while the approach was not perfect, it was fixed 
Seriously most of the complaints on the timer is just them forgetting to renew certs, which is something that is so petty...

gg5489 said:

forgetting certs happens to EVERYONE. I mean even google: https://www.pcworld.com/article/426771/expired-google-certificate-temporarily-disrupts-gmail-service.html


You responded with: 

Yea but forgetting certs is a lame excuse
Certbot and cron
Done

I responded with:

Manjaro is operated mainly by donations, of which CDN77 donates their server infrastructure, which doesn't support certbot and requires them to manually renew and upload certs at time of renewal.
This is just petty, you're receiving the distro done by people who volunteer their free time, you're not paying for anything. You can complain about petty stuff like this when you're actually paying for a product and it is not being delivered.
And what if you forget it? So I forget to renew something like AutoDDU or 24HS-Automator's cert for a day and now those projects shouldn't exist due to the day of the year it was available with a warning? 
Ah yes, 0.27% ""failure"" rate is unacceptable 
Anyways, I am one person, and it is a community wiki, anyone else can comment on this, but I have yet to hear a solid argument for removing it.


MechanoPixel said:

I don't think there's a valid logical reason to skip it but I will say I don't really trust them with my donation money
Dismissing Jonathon (rest his soul) for properly enforcing the policies that they agreed on is fishy AF and hints at some deeper management/functional issues within the collective
It's a decent distro for the end users but personally I wouldn't donate money to them for fear of it being misused in some way




TLDR: While Manjaro is not a perfect distro (worst thing they've done is likely them dismissing Jonathon for doing his job), most of the points on the page aren't good arguments.

**Comment by IThundxr** [comment]
Seeing this won’t probs be merged I’ll be closing this but imo I feel Manjaro isn’t good but that’s my opinion
